tests/ops/fixtures/saas/datadog_fixtures.py

```diff
@@ -95,3 +95,70 @@ def datadog_dataset_config(
     )
     yield dataset
     dataset.delete(db=db)
+
+
+@pytest.fixture(scope='session')
+def generate_test_data_on_datadog(*args, **kwargs):
+    import requests
+    import json
+
+    url = "https://http-intake.logs.datadoghq.com/api/v2/logs"
+
+    payload = json.dumps([
+        {
+            "ddsource": "nginx",
+            "ddtags": "env:staging,version:5.1",
+            "hostname": "i-012345678",
+            "message": "2019-11-19T14:37:58,995 INFO [process.name][20081] Hello noonari@xyz.com",
+            "service": "payment"
+        },
+        {
+            "ddsource": "nginx",
+            "ddtags": "env:staging,version:5.1",
+            "hostname": "i-012345678",
+            "message": "2019-11-19T14:37:58,995 INFO [process.name][20081] Hello noonari@xyz.com",
+            "service": "payment"
+        },
+        {
+            "ddsource": "nginx",
+            "ddtags": "env:staging,version:5.1",
+            "hostname": "i-012345678",
+            "message": "2019-11-19T14:37:58,995 INFO [process.name][20081] Hello noonari@xyz.com",
+            "service": "payment"
+        },
+        {
+            "ddsource": "nginx",
+            "ddtags": "env:staging,version:5.1",
+            "hostname": "i-012345678",
+            "message": "2019-11-19T14:37:58,995 INFO [process.name][20081] Hello noonari@xyz.com",
+            "service": "payment"
+        },
+        {
+            "ddsource": "nginx",
+            "ddtags": "env:staging,version:5.1",
+            "hostname": "i-012345678",
+            "message": "2019-11-19T14:37:58,995 INFO [process.name][20081] Hello noonari@xyz.com",
+            "service": "payment"
+        },
+        {
+            "ddsource": "nginx",
+            "ddtags": "env:staging,version:5.1",
+            "hostname": "i-012345678",
+            "message": "2019-11-19T14:37:58,995 INFO [process.name][20081] Hello noonari@xyz.com",
+            "service": "payment"
+        },
+        {
+            "ddsource": "nginx",
+            "ddtags": "env:staging,version:5.1",
+            "hostname": "i-012345678",
+            "message": "2019-11-19T14:37:58,995 INFO [process.name][20081] Hello noonari@xyz.com",
+            "service": "payment"
+        }
+    ])
+    headers = {
+        'Content-Type': 'application/json',
+        'Accept': 'application/json',
+        'DD-API-KEY': ''
```

Comment:
If you change the params to include the secrets fixture
```
def datadog_access_data(datadog_secrets):
```
you can access the API key like this
```
datadog_secrets["api_key"]
```